toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -36,6 +40,19 @@ LABEL=CDROM
 
 if grep -qs $ISO_ROOT /proc/mounts; then
     echo ISO root already mounted
+# Read from /proc/cmdline to get image configs from PXE server
+elif grep -q "image-config" $CMDLINE; then
+    for argument in $(cat "$CMDLINE")
+    do
+        IFS='=' read -r FLAG VALUE <<< $argument
+        if [ "${FLAG}" == "--image-config" ]; then
+            SERVER_CONFIG_PATH=$VALUE
+            IFS='/' read -ra CUT_DIRS <<< $VALUE
+            wget -r --no-parent -nH --cut-dirs=$((${#CUT_DIRS[@]}-4)) --directory-prefix="$ISO_ROOT" --reject="index.html*" "$SERVER_CONFIG_PATH"/
+            # Enable all scripts under config directory to be executable
+            find "$CONFIG_ROOT"/ -type f -iname "*.sh" -exec chmod a+x {} \;
+        fi
+    done
```

Comment:
Btw. I now just realized you only care about the value for `--image-config`, so you might be able to simplify a bit:
```suggestion
    IMAGE_CONFIG_VALUE=$(grep -oP "(?<=--image-config=)\S+" "$CMDLINE")
    IFS='/' read -ra CUT_DIRS <<< $IMAGE_CONFIG_VALUE
    wget -r --no-parent -nH --cut-dirs=$((${#CUT_DIRS[@]}-4)) --directory-prefix="$ISO_ROOT" --reject="index.html*" "$IMAGE_CONFIG_VALUE"/
    # Enable all scripts under config directory to be executable
    find "$CONFIG_ROOT"/ -type f -iname "*.sh" -exec chmod a+x {} \;
```

I'm not entirely sure what you need from `CUT_DIRS`, but there might be some simplification possible there as well.

And shouldn't the scripts inside `CONFIG_ROOT` be already executable? I'm curious, if we don't have a bug elsewhere.